packages/lexical-editor/src/components/Toolbar/Toolbar.tsx

```diff
@@ -30,15 +27,50 @@ interface FloatingToolbarProps {
 
 const FloatingToolbar: FC<FloatingToolbarProps> = ({ children, type, anchorElem, editor }) => {
     const popupCharStylesEditorRef = useRef<HTMLDivElement | null>(null);
-    const { toolbarType, setToolbarType, setTextBlockSelection } = useRichTextEditor();
-    const [activeEditor, setActiveEditor] = useState(editor);
+    const { toolbarType, setToolbarType } = useRichTextEditor();
 
     useEffect(() => {
         if (toolbarType !== type) {
             setToolbarType(type);
         }
     }, [type]);
 
+    const mouseMoveListener = useCallback((e: MouseEvent) => {
+        if (popupCharStylesEditorRef?.current && (e.buttons === 1 || e.buttons === 3)) {
```

Comment:
I found what these magic numbers are :D, added the comments as well. Because is the official standard I left them. 
[Mozzila docs link](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons)
Numbers:
1 = The left button
2 = The right button
4 = The middle button
8 = The fourth button (Browser Back)
16 = The fifth button (Browser Forward)

If you click for example first and second buttons in the same time on the mouse in that case you will get 3 or 1 + 2 = 3 
I didn't know that this exist. :)